db/migrate/20191119111806_split_disk_type_to_thin_and_format_for_disk.rb

```diff
@@ -0,0 +1,17 @@
+class SplitDiskTypeToThinAndFormatForDisk < ActiveRecord::Migration[5.1]
+  def up
+    add_column :disks, :format, :string
+    add_column :disks, :thin, :boolean
+    Disk.connection.execute("UPDATE disks SET thin = (CASE disk_type WHEN 'thin' THEN true ELSE false END) WHERE disk_type IN ('thin', 'thick')")
```

Comment:
You can define a Disk subclass in this migration e.g. `class Disk < ActiveRecord::Base; end`

Reply:
Probably something like `Disk.in_my_region.where(:disk_type => 'thin').update_all(:thin => true)` would work
For the format you probably need to look at the EMS type to set the format e.g. vmdk/qcow/etc...